### PR TITLE
Minor refactoring and reformatting

### PR DESCRIPTION
**Pull request by ConnorSkees**
- Tighten whitespace
- Remove superfluous `assert` statement
- Reorder imports
- Make parameters required and remove default values

### CODE REVIEW

**Comment by Nodraak** [comment]
Thanks to `if __name__ == "__main__":`, the script can be run standalone, without pip install-ing it. Maybe you could should the README to let know people of this new feature?

File: deletefb/deletefb.py [hunk 1]
```diff
@@ -1,54 +1,57 @@
...
     """
-
-    assert all((user_email_address,
-                user_password,
-                user_profile_url)), "Insufficient arguments provided"
-
     # The Chrome driver is required because Gecko was having issues
     chrome_options = Options()
-    prefs = {"profile.default_content_setting_values.notifications" : 2}
+    prefs = {"profile.default_content_setting_values.notifications": 2, 'disk-cache-size': 4096}
```
Comment (weskerfoot) [inline deletefb/deletefb.py]: I'm assuming the `disk-cache-size` is necessary to persist sessions across runs? That's useful.

**Comment by weskerfoot** [comment]
Can you fix the conflicts? I think there are some useful things in here that weren't in your other PR

**Comment by ConnorSkees** [comment]
I fixed the conflicts and squashed the commits :-) I think it's good to go

**Pull request merged**